madminer/utils/interfaces/mg.py

```diff
@@ -507,12 +491,9 @@ def setup_mg_reweighting_with_scripts(
 
     #  Card copying commands
     if reweight_card_file_from_mgprocdir is not None:
-        copy_commands = "cp {}/{} {}{}\n".format(
-            mg_process_directory_placeholder,
-            reweight_card_file_from_mgprocdir,
-            mg_process_directory_placeholder,
-            "/Cards/reweight_card.dat",
-        )
+        copy_commands = f"cp " \
```

Comment:
Definitely better than before, but maybe this is even better:

```
copy_commands = (
    f"cp {mg_process_directory_placeholder}/{reweight_card_file_from_mgprocdir}"
    f" {mg_process_directory_placeholder}/Cards/reweight_card.dat\n"
)
```

[PEP8 says: "Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation."](https://www.python.org/dev/peps/pep-0008/)

Reply:
Ok. Changing it to `( ... )` style.